v2/pkg/clusterresources/clusterresources_test.go

```diff
@@ -0,0 +1,176 @@
+package clusterresources
```

Comment:
@sherine-k - Nice one coverage  is at 82.7%

using 

```
go tool cover -html=tests/results/cover.out -o tests/results/cover.html
```
just for interest sake